renderdoc/replay/replay_driver.cpp

```diff
@@ -565,7 +565,7 @@ bool HighlightCache::FetchHighlightPositions(const MeshDisplay &cfg, FloatVector
   {
     uint32_t dim = PatchList_Count(cfg.position.topo);
 
-    uint32_t v0 = uint32_t(idx / dim) * dim;
+    uint32_t v0 = dim == 0 ? 0 : (uint32_t(idx / dim) * dim);
```

Comment:
Isn't the problem here that `PatchList_Count` returned 0 at all for a valid patchlist topology? It looks like when I implemented it I didn't do `+ 1` when subtracting from `Topology::PatchList_1CPs` so all of them are off-by-one.

It's a valid bug, but more wide-reaching than just this fix!